### PR TITLE
stop consignee sending in ie815

### PR DESCRIPTION
**Pull request by ChrisCookOC**

### CODE REVIEW

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala
Comment (mpbostock) [inline it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala:1]: Is it worth adding another test here where the consignor is empty in the XML. We should probably catch this case before we send down to EIS. Is this a Bad Request or Forbidden though 🤔 ?

Comment (ChrisCookOC) [reply it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala:1]: Will add. If field is present but blank we get a 403 (since we aren't logged in as the blank ern), if xml is missing the field totally it will be 400

File: it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala [hunk 1]
```diff
@@ -147,6 +147,12 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
       postRequest(IE815).status mustBe FORBIDDEN
     }
 
+    "return forbidden (403) when the consignor is empty" in {
+      withAuthorizedTrader(consigneeId)
```
Comment (mpbostock) [inline it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala:151]: This could be the consignorId as the consignor should be submitting (the consignee is covered in the other test)

File: test/uk/gov/hmrc/excisemovementcontrolsystemapi/data/TestXml.scala [hunk 2]
```diff
@@ -128,6 +125,113 @@ trait TestXml {
     </urn:Body>
   </urn:IE815>
 
+  lazy val IE815NoConsignor: Elem = <urn:IE815 xmlns:urn="urn:publicid:-:EC:DGTAXUD:EMCS:PHASE4:IE815:V3.01"
+                                    xmlns:urn1="urn:publicid:-:EC:DGTAXUD:EMCS:PHASE4:TMS:V3.01"
+                                    xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope"
+                                    xmlns="http://www.hmrc.gov.uk/ChRIS/Service/Control"
+                                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <urn:Header>
```
Comment (mpbostock) [inline test/uk/gov/hmrc/excisemovementcontrolsystemapi/data/TestXml.scala:133]: could we maybe add a method "getIE815" here that takes a consignorID in and returns the XML. This method can then be re-used by the two vals "IE815 and IE815NoConsignor" to get the xml and then it's only defined once?

**Review by mpbostock — APPROVED** [review]
